signac/contrib/project.py

```diff
@@ -434,6 +436,8 @@ def build_job_search_index(self, index, _trust=False):
         """
         return JobSearchIndex(index=index, _trust=_trust)
 
+    @deprecated(deprecated_in="1.3", removed_in="2.0", current_version=__version__,
+                details="Use the detect_schema function instead.")
```

Comment:
For consistency with the other deprecation details strings, add `()` to `detect_schema`.